simple_history/manager.py

```diff
@@ -126,7 +127,8 @@ def bulk_history_create(
                 default_user or self.model.get_default_history_user(instance),
             )
             row = self.model(
-                history_date=getattr(instance, "_history_date", timezone.now()),
+                history_date=default_date
+                or getattr(instance, "_history_date", timezone.now()),
```

Comment:
Would prefer that this follow the pattern of `_history_user`, such that `_history_date` takes precedence over `default_date`

Reply:
Ah true, `_history_date` gives more fine-grained control so it should take precendence. Haven't really used it yet. :)